protos/bottle/account/v1.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+import "bottle/account/v1/service.proto";
+
+package bottle.account;
+
+service V1 {
+  rpc NotificationMethod(bottle.account.v1.NotificationMethodRequest) returns (bottle.account.v1.NotificationMethodResponse);
```

Comment:
I'm wonder if we should just return all of their notification preferences at once but we can deal with that later :+1:

Reply:
I think this is good for now. We can update it later. One thing that I was thinking was also maybe moving notification preferences into bullhorn and have _it_ keep track of everything, but I'd rather just get this to a working state first.